test/AutoRest.TestServerLowLevel.Tests/lro.cs

```diff
@@ -23,7 +23,7 @@ public class LroTest : TestServerLowLevelTestBase
             var options = new AutoRestLongRunningOperationTestServiceClientOptions();
             options.AddPolicy(new CustomClientRequestIdPolicy(), HttpPipelinePosition.PerCall);
             var value = RequestContent.Create(new object());
-            var operation = await new LROsCustomHeaderClient(Key, endpoint, options).PostAsyncRetrySucceededAsync(value);
+            var operation = await new LROsCustomHeaderClient(Key, endpoint, options).PostAsyncRetrySucceededAsync(false, value);
```

Comment:
Would it make sense to add some cases where we pass true to an LRO wait method?

Reply:
All lro tests now cover both cases